docs/ql-style-guide.md

```diff
@@ -335,7 +346,8 @@ deprecated Expr getInitializer()
 ```
 
 ```ql
-  if e1 instanceof BitwiseOrExpr or e1 instanceof LogicalOrExpr then (
+  if e1 instanceof BitwiseOrExpr or e1 instanceof LogicalOrExpr
+  then (
```

Comment:
I don't mind this but I thought the consensus was `then` at the end of the line?

Reply:
I believe the consensus was to allow both.  With the autoformatter it makes sense to choose one of the two options.  (And IIRC I championed the "`then` at eol" style)